test/integration/user_can_view_past_orders_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+class UserCanViewPastOrdersTest < ActionDispatch::IntegrationTest
+  test "user can view past orders" do
+    category_1 = Oil.create(name: "Lard")
+    item_1 = Chip.create(name: "Slotachips", price: 20, description: "Super yummy", oil_id: category_1.id)
+    user = User.create(username: "John", password: "Password")
+    order = user.orders.create(total_price: 20)
+    order.chip_orders.create(chip_id: item_1.id, quantity: 1, subtotal: 20)
+
+    visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.